serving/docs/modes.md

```diff
@@ -65,6 +65,10 @@ and python handler functions.
 - `gpu.maxWorkers`: Maximum number of workers for GPU.
 - `cpu.minWorkers`: Minimum number of workers for CPU.
 - `cpu.maxWorkers`: Maximum number of workers for CPU.
+- `required_memory_mb`: Specify the required memory (CPU and GPU) in MB to load the model.
```

Comment:
For Python based engines, are we assuming that this accounts for the number of mpi workers? Or should we account for mpi workers and parallel_loading in the estimate calculation?